qkeras/qlayers.py

```diff
@@ -57,6 +57,7 @@
 from tensorflow.keras.layers import InputSpec
 from tensorflow.keras.layers import Layer
 from tensorflow.keras.models import model_from_json
+from tensorflow.keras.utils import deserialize_keras_object, get_custom_objects
```

Comment:
For consistency, could you seperate this line to two lines

from tensorflow.keras.utils import deserialize_keras_object
from tensorflow.keras.utils import get_custom_objects

Reply:
Do you want me to fix this one before we restructure the modules in a separate PR?

Reply:
No, thanks. Please go ahead with another PR. I will let you know what I need if anything coming up about this PR (I am checking the PR in the internal system).